enterprise/frontend/src/embedding-sdk/lib/interactive-question/load-question.ts

```diff
@@ -57,11 +51,5 @@ export const runQuestionOnLoadSdk =
       question = question.setParameterValues(parameterValues);
     }
 
-    const result = await runQuestionQuerySdk({
-      question,
-      originalQuestion,
-      cancelDeferred,
-    });
-
-    return { ...result, originalQuestion };
+    return { question, originalQuestion };
```

Comment:
I only moved the run query part out of this function so I could update the state with the loaded question first, before the query is running. This function is only used in one place, so it's safe to make this change.